easybuild/easyconfigs/l/LAPACK/LAPACK-3.8.0-gompi-2019a.eb

```diff
@@ -0,0 +1,16 @@
+name = 'LAPACK'
+version = "3.8.0"
+
+homepage = 'http://www.netlib.org/lapack/'
+description = """LAPACK is written in Fortran90 and provides routines for solving systems of
+ simultaneous linear equations, least-squares solutions of linear systems of equations, eigenvalue
+ problems, and singular value problems."""
+
+toolchain = {'name': 'gompi', 'version': '2019a'}
```

Comment:
hm... I updated an existing easyconfig but I don't see why `LAPACK` should be at `gompi` instead of `GCC` level...

Reply:
Indeed @migueldiascosta please look at #6615.

Reply:
hm, now that you mention that PR, I recall having seen it before, should have looked for it...